tests/Agent/IntegrationTests/IntegrationTestHelpers/AgentLogBase.cs

```diff
@@ -17,7 +17,7 @@ namespace NewRelic.Agent.IntegrationTestHelpers
 {
     public abstract class AgentLogBase
     {
-        public const string LogLineContextDataRegex = @"\[pid: \d+, tid: \d+\] ";
+        public const string LogLineContextDataRegex = @"\[pid: \d+, tid: [\w-#:(,\/.)]+\] ";
```

Comment:
Thanks for explaining what is going on with this RegEx.  I do have a suggestion: I don't think we care too much, at this point, about being very precise in what this particular regex is matching.  Therefore, I think the regex could be simplified to:
`\[pid: .+, tid: .+\]`
in other words, match one or more of _any_ character to the right of the `pid:` and `tid:` 'fields'.

Alternatively, if we want to continue to be as selective as possible in what kind of text we will match for the thread id, I think it would be good to add a comment above this line with an example of what kind of text it's looking to match (like you have in the PR description).

Reply:
I'm ok with this approach for `tid`, but I'd like to leave `pid` as `\d`. Will submit commit.